### DISCUSSION TITLE
How to get destination directory name/path within jinja2 templates

### DISCUSSION THREAD

**Discussion by umanamente** [copier-org/copier#1624] [Q&A]
I want to create a copier template for a Django App (not the project, but for each individual app within it). Since the project will contain multiple apps, I've placed the template files in the root of the copier template. However, I'm facing an issue where I want to use the app name within the templates. Creating a specific question for the app name seems redundant because the app name is already specified as a destination in the `copier copy` command. 
Is there a way to access the destination path within Jinja2 templates?

**Comment by sisp** [comment]
This is currently not possible but has been discussed before in #1050. We're open to a contribution in case you're interested. 🙂

Duplicate of #1623.
